rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -41,7 +41,8 @@
 
 SMALLEST_AIRDROP_SIZE: Final = 20900
 AIRDROPS_REPO_BASE: Final = f'https://raw.githubusercontent.com/rotki/data/{"main" if is_production() else "develop"}'  # noqa: E501
-AIRDROPS_INDEX: Final = f'{AIRDROPS_REPO_BASE}/airdrops/index_v1.json'
+AIRDROPS_INDEX_VERSION: Final = '1'
+AIRDROPS_INDEX: Final = f'{AIRDROPS_REPO_BASE}/airdrops/index_v{AIRDROPS_INDEX_VERSION}.json'
```

Comment:
Why is this a constant here? I don't understand. Can't the index be different if pulled from remote?